blurb/blurb.py

```diff
@@ -486,6 +486,9 @@ def finish_entry():
                 # we'll complain about the *first* error
                 # we see in the blurb file, which is a
                 # better user experience.
+                if key == "gh-issue" and int(value) < 32426:
+                    throw("The gh-issue number should be 32426 or above.")
```

Comment:
Maybe something like this? It gives a hint they may be using the PR number instead of issue number.

```suggestion
                    throw("The gh-issue number must be 32426 or above, not a PR number.")
```